packages/nvidia_nat_nemo_customizer/tests/test_nemo_customizer.py

```
@@ -0,0 +1,1121 @@
+            customization_config="meta/llama-3.2-1b-instruct@v1.0.0+A100",
+        )
+
+        assert config.entity_host == "https://nmp.example.com"
+        assert config.datastore_host == "https://datastore.example.com"
+
+    def test_full_config(self):
+        """Test full configuration with all options."""
+        config = NeMoCustomizerTrainerAdapterConfig(
+            entity_host="https://nmp.example.com",
+            datastore_host="https://datastore.example.com",
+            hf_token="my-token",
```

Ruff (0.14.8)

Possible hardcoded password assigned to argument: "hf_token"

(S106)